docs/developer/checkout.mdx

```diff
@@ -1147,9 +1190,146 @@ As a result, we get an order object:
 }
 ```
 
+### Creating transaction
+:::info
+
+This feature was introduced in **Saleor 3.3**.
+
+:::
+
+:::caution
+
+This feature is currently in **Feature Preview**. This part of Saleor is not complete
+and subject to change but is available to experiment and provide feedback.
+
+:::
+
+Transaction stores details of payment transaction attached to order or checkout.
+
+
+The [transactionCreate](developer/api-reference/mutations/transaction-create.mdx) mutation takes the following arguments:
+- `id`: The ID of the checkout or order.
+- `transaction`: Input data required to create a new transaction object.
+- `transactionEvent`: Data that defines a transaction event.
```

Comment:
It was hard for me to understand what the Event for, before reading the example.